internal/license/integration/BUILD.bazel

```
@@ -0,0 +1,14 @@
+load("//bazel/go:go_test.bzl", "go_test")
+
+go_test(
+    name = "integration_test",
+    srcs = ["license_integration_test.go"],
+    tags = [
+        "integration",
+        "requires-network",
+    ],
+    deps = [
+        "//internal/license",
+        "@com_github_stretchr_testify//assert",
+    ],
+)
```

```suggestion
go_test(
    name = "integration_test",
    srcs = ["license_integration_test.go"],
    gotags = ["integration"],
    tags = [
        "integration",
        "requires-network",
    ],
    deps = [
        "//internal/license",
        "@com_github_stretchr_testify//assert",
    ],
)
```

We need to tell rules_go to set the go build tag for this test. Alternatively, we could set this globally in the `.bazelrc`, but I'm not sure if this would break the other unfinished integration tests.

---

The idea was to implicitly set the gotag when the integration config is used. That way, I'm not forced to keep tags and gotags for individual targets at sync. Now, if don't want to use the integration gotag in future tests, I agree that it makes sense to list it here. But then the question is if we need it in the first place: if it's only there to exclude the test from `blaze test //...`, we can just remove it in this commit.

---

Oh. I totally missed the gotags in the `integration` and `integration-only` configs in bazelrc.
Your solution is better.